crates/ruff_linter/src/rules/refurb/rules/print_empty_string.rs

```diff
@@ -30,6 +30,11 @@ use crate::checkers::ast::Checker;
 /// print()
 /// ```
 ///
+/// ## Fix safety
+/// This fix is unsafe because if the removed argument or keyword is an
+/// expression with side effects. Removing such arguments may
+/// skip the execution of those side effects.
```

Comment:
How does this sound? I think the rule only looks for `""` (empty string) positional arguments, so the unsafety is only applied to the `sep` keyword argument, as far as I can tell. This one is also only sometimes unsafe, so I tried to reflect that in the beginning of the first sentence.

```suggestion
/// This fix is marked as unsafe if it removes an unused `sep` keyword argument
/// that may have side effects. Removing such arguments may change the program's
/// behavior by skipping the execution of those side effects.
```